docs/src/components/about/Team.tsx

```diff
@@ -0,0 +1,390 @@
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import Grid from '@mui/material/Grid';
+import Paper, { PaperProps } from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Tooltip from '@mui/material/Tooltip';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import Link from 'docs/src/modules/components/Link';
+import ROUTES from 'docs/src/route';
+import Section from 'docs/src/layouts/Section';
+import GradientText from 'docs/src/components/typography/GradientText';
+import teamMembers from 'docs/data/about/teamMembers.json';
+
+interface Profile {
+  name: string;
+  /**
+   * Role, what are you working on?
+   */
+  title: string;
+  /**
+   * Country where you live in, ISO 3166-1.
+   */
+  locationCountry: string; // https://flagpedia.net/download/api
+  /**
+   * Image URL.
+   */
+  src?: string;
+  /**
+   * Lives in.
+   */
+  location?: string;
+  /**
+   * Short summary about you.
+   */
+  about?: string;
+  github?: string;
+  twitter?: string;
+}
+
+function Person(props: Profile & { sx?: PaperProps['sx'] }) {
+  return (
+    <Paper variant="outlined" sx={{ p: 2, height: '100%', ...props.sx }}>
```

Comment:
```suggestion
    <Paper variant="outlined" sx={{ p: 2, height: '100%', ...props.sx }}>
```

To be carefull with spreading the `sx` prop like this, as it may be a function too.

Reply:
Gotcha! That was this way before ⎯ should I do something with it?

Reply:
Let's leave it for now, we have other occurrences of spreading in the docs. In the end these are internal components. Ideally we should handle this similar to: https://github.com/mui/material-ui/blob/master/docs/src/components/action/ArrowButton.tsx#L48

Reply:
Sweet, alright ⎯ thanks for the clarification! I'll see if I can change some of these already! 🤙